# Lab | Inferential statistics
### Instructions
It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

Set up the hypothesis test.
Write down all the steps followed for setting up the test.
Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.
If you finished the previous question, please go through the code for principal_component_analysis_example provided in the files_for_lab folder .

**Step 1:** Define the null hypothesis - This is our assumption about the population. It is defined by H0 and in this case **H0: μ = 120**;

**Step 2:** Define the alternative hypothesis - This means, what if our assumption is not true. It is defined by Ha and in this case **Ha: μ <> 113.1**. 

**Step 3:** Determine if it is a one-tailed or a two-tailed test. This is a **two-tailed test**.

**Step 4:** N = 100, we can either use z-test or t-test.

**Step 5:** Level of significance: This defines the rejection region/critical region, it's the probability of making the wrong decision when the null hypothesis is true. Usually it is 0.05. It is defined by greek letter 'alpha'. In the medical field this would go down to 0.01.
 
**Step 6:** Calculate the test statistic based on the given information.

**Step 7:** Check the table to determine the critical value.
<br> For z-test you have fixed values according to Confidence Level.
<br> For t-test you have to calculate according to the degrees of freedom (df), which is the *sample_size - 1*.

**Step 8:** Make conclusions:
* If the test statistic falls in the critical region, then we reject the Null Hypothesis
* If the test statistic falls in the region between the critical region, then we fail to reject the Null Hypothesis.

In [1]:
import math
import scipy.stats

sample_mean = 113.1
pop_mean = 120
pop_std = 21.21
n = 100

statistic = (sample_mean - pop_mean)/(pop_std/math.sqrt(n))
statistic

-3.253182461103256

In [4]:
# Determining the p-value
p_val = scipy.stats.t.sf(abs(statistic), df = n-1) 
print("p-value: ", p_val)

p-value:  0.0007807351304365829


### Conclusion

> statistic = -3.25 

> p-value = 0.00018

- From the result, we rejected the H0, meaning

    **Blood pressure of sample group in Honolulu is significantly different from the regular population**